default.nix

```diff
@@ -425,14 +447,12 @@ in rec {
     buildPhase = ''
       source ${./nix/expectShFunctions.sh}
 
-      expectFilePresent ./node_modules/.yarn-integrity
-
       # check dependencies are installed
-      expectFilePresent ./node_modules/@yarnpkg/lockfile/package.json
+      expectFilePresent ./deps/yarn2nix/node_modules/@yarnpkg/lockfile/package.json
```

Comment:
These expectations were assuming yarn2nix's dependencies being hoisted outside of the yarn2nix dir, which has been explicitly removed here.